src/iiif/tools.py

```diff
@@ -56,13 +71,17 @@ def get_info_from_iiif_url(iiif_url):
 
     ## WABO
     # iiif_url = \
-    # "https://acc.images.data.amsterdam.nl/iiif/2/wabo:SDZ-38657-4900487_628547"
+    # "https://acc.images.data.amsterdam.nl/iiif/2/wabo:SDZ-38657-4900487_628547/full/full/0/default.jpg""
     # SDZ=stadsdeel  38657=dossier  4900487=olo_liaan_nummer  628547=document_barcode
 
+    # At the end of the url, this can be appended '?source_file=true', which means we'll bypass 
+    # cantaloupe and go directly for the source file
+
     try:
         source = iiif_url.split(':')[0].split('/')[1]  # "edepot" or "wabo"
         relevant_url_part = iiif_url.split(':')[1].split('/')[0]
-        formatting = iiif_url.split(':')[1].split('/', 1)[1] if '/' in iiif_url.split(':')[1] else ''
+        formatting = iiif_url.split(':')[1].split('/', 1)[1].split('?')[0] if '/' in iiif_url.split(':')[1] else ''
```

Comment:
yooooo

Reply:
wat is dit? xD

Reply:
Daar zit haast achter ja. Ga ik oplossen in een opschoning.